src/pages/Messenger/components/Chat/interfaces.ts

```diff
@@ -0,0 +1,25 @@
+export interface IChatProps {
+    id: number,
+    title: string,
+    avatar: null | string,
+    created_by: number,
+    unread_count: number,
+    last_message: {
+        id: number,
+        content: string,
+        time: string
+    },
+};
+
+export interface IChatList {
+    id: number,
+    title: string,
+    avatar: null | string,
+    created_by: number,
+    unread_count: number,
+    last_message: {
+        id: number,
+        content: string,
+        time: string
```

Comment:
Удобнее будет работать с объектом типа Date
```suggestion
        time: Date
```